### ISSUE TITLE
Bindings for Haskell?

### ISSUE THREAD

**Issue by agocorona** [kelseyhightower/nocode#3472]

**Comment by OverclockedSanic** [comment]
Yeah good idea. In fact we should rewrite the entire thing in haskell, that way no one will run this code, guaranteed.

**Comment by austindd** [comment]
I already did not submit a PR to add Haskell bindings. You can find it not listed in the Pull Requests. Feel free to add comments here if you have any feedback on it.
